src/main/java/mate/jdbc/model/Manufacturer.java

```diff
@@ -0,0 +1,45 @@
+package mate.jdbc.model;
+
+public class Manufacturer {
+    private Long id;
+    private String name;
+    private String country;
+
+    public Manufacturer() {
+    }
+
+    public Manufacturer(String name, String country) {
+        this.name = name;
+        this.country = country;
+    }
+
+    public Manufacturer(Long id, String name, String country) {
+        this.id = id;
+        this.name = name;
+        this.country = country;
```

Comment:
use another constructor to not duplicate your code
```suggestion
        this(name, country);
        this.id = id;
```